ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -2819,6 +2819,38 @@ def test_jax_remainder(
     )
 
 
+# rint
+@handle_frontend_test(
+    fn_tree="jax.numpy.rint",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
I think you should change this to `helpers.get_dtypes("valid")` since the input can also be of type `int` or `bool`. Eg: 

```python
np.rint(np.asarray([True, False]))
>>> array([1., 0.], dtype=float16)
```